src/main/scala/com/lucidchart/open/relate/SqlResult.scala

```diff
@@ -239,13 +242,15 @@ class SqlResult(val resultSet: java.sql.ResultSet) {
     }
   }
 
+  @deprecated("Use long(String) or string(String) instead", "1.8.0")
```

Comment:
mysql requires scala/java bigint for BIGINT UNSIGNED. Because of this, mysql will use bigints for ALL autoincrement columns. We can't deprecate this for longs.